updated checkAttrResponsive, updated checkAttr (#95)
infinum/eightshift-docs: website/docs/basics/helpers-php.md

## checkAttr

> This function is the really similar to the same JavaScript function on this [link](helpers-javascript). Except there is no last parameter which is used only in wp-admin to detect `undefined` attributes.

Reviewer: ```suggestion
This function is the PHP equivalent of [this JavaScript function](helpers-javascript), except there is no`undefinedAllowed` parameter, which is used only on the WordPress admin side.
```

## checkAttrResponsive

> This function is also the really similar to the same JavaScript function on this [link](helpers-javascript). Except there is no last parameter which is used only in wp-admin to detect `undefined` attributes.

Reviewer: ```suggestion
This function is the PHP equivalent of [this JavaScript function](helpers-javascript), except there is no `undefinedAllowed` parameter - it's used only on the WordPress admin side.
```

Author: >Last isn't the best word here, as more parameters could be added in the future :)

I fully agree! 👍
